dvc/repo/experiments/push.py

```diff
@@ -32,28 +31,26 @@ def __init__(self, msg, result):
 def notify_refs_to_studio(
     repo: "Repo", git_remote: str, **refs: list[str]
 ) -> Optional[str]:
-    import os
+    from dvc_studio_client.config import get_studio_config
```

Comment:
Let's not change `notify_refs_to_studio` please. 

`dvc_studio_client` does a lot of things that we don't care about. Eg: it returns an empty dictionary when it cannot figure out a git remote url, which will make us skip the push, instead of raising an error.  It also opens a git `Repo`, which is wasteful.
And all the logging messages don't make sense for `exp push`.

I'd rather prefer DVC to be in control of its own config.

Secondly, `exp push` does not need to support any `exp.git_remote` or `DVC_EXP_GIT_REMOTE` as `git_remote` is always required. So, it's not worth it complicating a straight-forward one-liner code with an indirection to `get_repo_url`.